collaboration/neighborhood.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python3
+#!/usr/bin/python
```

Comment:
I don't know why you are changing the #! from a version-specific call to a version-agnostic.  We know that most systems being used now bind /usr/bin/python to Python 2.  Would /usr/bin/env be better here?

Reply:
Yes, the change is unneeded. I have updated all the shebangs